website/content/docs/security/acl/acl-multiple-datacenters.mdx

```diff
@@ -0,0 +1,234 @@
+---
+layout: docs
+page_title: ACL in Multiple Datacenters
+description: >-
+  When Consul ACLs are enabled in multiple datacenters that are federated together,
+  specific bootstrapping of ACLs and policies are needed
+---
+
+# ACLs in Federated Datacenters
+
+This topic describes how to set up Consul's access control list (ACL) system
+in cluster deployments that span multiple data centers. This documentation is applicable
+to new clusters rather than existing clusters.
+
+-> **1.4.0 and later:** This guide only applies in Consul versions 1.4.0 and later.
+
+## Configure ACLs in the Primary Datacenter
+
+In a federated Consul deployment, one of the datacenters is marked as the primary datacenter.
```

Comment:
Would it be clear to a new user what we mean by "federated deployment"? Should we provide a quick definition or is it reasonable to expect that they understand what we mean if they've navigated to this page?